hack/generator/pkg/astmodel/property_injector.go

```diff
@@ -25,19 +27,34 @@ func NewPropertyInjector() *PropertyInjector {
 
 // Inject modifies the passed type definition by injecting the passed property
 func (pi *PropertyInjector) Inject(def TypeDefinition, prop *PropertyDefinition) (TypeDefinition, error) {
-	return pi.visitor.VisitDefinition(def, prop)
+	result, err := pi.visitor.VisitDefinition(def, prop)
+	if err != nil {
+		return TypeDefinition{}, errors.Wrapf(err, "failed to inject property into %q", def.Name())
```

Comment:
Maybe include the name of the property too (in case the error has another cause)?
```suggestion
		return TypeDefinition{}, errors.Wrapf(err, "failed to inject property %q into %q", prop.Name(), def.Name())
```

Reply:
Fixed